workitem_test.go

```diff
@@ -110,16 +109,16 @@ func TestListByFields(t *testing.T) {
 	repo := models.NewWorkItemRepository(ts)
 	controller := WorkitemController{ts: ts, wiRepository: repo}
 	payload := app.CreateWorkitemPayload{
-		Name: "ListByName Name",
 		Type: "1",
 		Fields: map[string]interface{}{
+			"system.title" : "run integration test",
 			"system.owner": "aslak",
```

Comment:
The `owner` field has been renamed to `assignee` in #185. Remember to change it once this gets merged.